analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -592,11 +593,13 @@ private Map<RelaxedProjectVersionRef, ProjectVersionRef> getDependencies(Project
                 }
                 Set<ResolvedDependency> target;
                 if (internalConfig.overrideTransitive() == null || !internalConfig.overrideTransitive()) {
```

Comment:
This is the original code, but it looks like this outer `if` was never necessary?